### PR TITLE
chore: Backport "all artifactory endpoints on master are switched to lf" to v1.8

### LINKED ISSUE #14581: v1.7 and v1.8 are using the lf artifactory for installing magma
### 1.7, 1.8 using magma-packages

* [x] test 1.7, 1.8 install with sed script in install.sh
  * between checkout and install
  * 1.7 force merge needed
  * 1.8 OK from govind/yogesh needed
  * 1.7 needs proto fix (?)
* [x] merge this
  * [x] 1.7
  * [x] 1.8
* [x] communicating in #dev (?) that 1.7 and 1.8 install now use the new artifactory
* [x] asking lf for the virt repo

### PR DESCRIPTION
**Pull request by voisey**
## Summary

Backport of https://github.com/magma/magma/pull/14508 to v1.8.

## Test Plan

- [x] Tested that this branch can be installed on a blank Ubuntu box with `agw_install_ubuntu.sh` @ajahl 
- [x] Tested that magma_deb can be brought up on this branch and that basic integration tests run @sebathomas

### CODE REVIEW

File: orc8r/tools/packer/scripts/ansible_debian.sh [hunk 1]
```diff
@@ -17,4 +17,4 @@ bash -c 'echo -e "deb http://ppa.launchpad.net/ansible/ansible/ubuntu trusty mai
 
 sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 93C4A3FD7BB9C367
 sudo apt-get update
-sudo apt-get install -y ansible
+sudo apt-get install -y ansible
```
Comment (sebathomas) [inline orc8r/tools/packer/scripts/ansible_debian.sh]: Nitpick: unneeded change.

Comment (voisey) [reply orc8r/tools/packer/scripts/ansible_debian.sh]: Removed, thanks!

**Review by sebathomas — APPROVED** [review]

**Review by nstng — APPROVED** [review]

**Review by MoritzThomasHuebner — APPROVED** [review]

**Review by wolfseb — APPROVED** [review]
LGTM

**Review by xbend — APPROVED** [review]

**Review by panyogesh — APPROVED** [review]
LGTM

**Pull request merged**